### DISCUSSION TITLE
Maven Dependency for 1.3.0 None Existent?

### DISCUSSION THREAD

**Discussion by crustycrabthe3rd** [obs-websocket-community-projects/obs-websocket-java#127] [Q&A] [answered]
# Describe the bug
I tried adding v1.3.0 to my project but it's saying the dependency doesn't exist, also looked on maven central, same thing!

**Comment by ChristopheCVB** [comment, marked as answer]
Yup, there's been a hiccup in the deployment of this version.
You can use the snapshot version instead:
https://oss.sonatype.org/content/repositories/snapshots/net/twasi/obs-websocket-java/1.3.0-SNAPSHOT/

**Comment by crustycrabthe3rd** [comment]
> Yup, there's been a hiccup in the deployment of this version.
> You can use the snapshot version instead:
> https://oss.sonatype.org/content/repositories/snapshots/net/twasi/obs-websocket-java/1.3.0-SNAPSHOT/

Thank you, i'm not quite versed in this, so sorry for asking, which file should i include in my intellij ide?

**Comment by Pjiesco** [comment]
> > Yup, there's been a hiccup in the deployment of this version.
> > You can use the snapshot version instead:
> > https://oss.sonatype.org/content/repositories/snapshots/net/twasi/obs-websocket-java/1.3.0-SNAPSHOT/
> 
> Thank you, i'm not quite versed in this, so sorry for asking, which file should i include in my intellij ide?

You don't need to download a file, just add `https://oss.sonatype.org/content/repositories/snapshots/` as a repository and the 1.3.0 snapshot should be available

**Comment by DorianSnowball** [comment]
Could you please add this solution to the Readme as well @Pjiesco ? 
Because right now you state in your Readme to download 1.3.0 from Mavencentral which WILL result in errors for every user that tries it (like me just now)

So please either fix the maven deploy or add it to the Readme.
Or at least keep this issue open so it's more visible.
